achievements.lua

```diff
@@ -118,12 +162,115 @@ local function export_data()
 	local data = achievements.gameData
 	json.encodeToFile(get_achievement_data_file_path(data.gameID), true, data)
 end
-function achievements.save()
+function achievements.save(to_file)
+	if to_file == nil then
+		to_file = "./" .. default_achievement_file_name
+	end
```

Comment:
See comment on _+150 to +154_. Additionally, we don't want multiple achievement files in the save data, which means that asking the developer to set a customized filename as an argument on each function call rather than as an overwriteable value configured at initialization time is bad practice.

Reply:
Done!